src/bnb_trading/weekly_tails.py

```
@@ -172,358 +174,394 @@
-            
+
         except Exception as e:
             logger.error(f"Грешка при изчисляване на силата на опашката: {e}")
             return 0.0
-    
-    def check_fib_tail_confluence(self, fib_levels: Dict[float, float], 
-                                 current_price: float, 
-                                 tails_analysis: List[Dict]) -> Dict[str, any]:
+
+    def check_fib_tail_confluence(
+        self, fib_levels: Dict[float, float], current_price: float, tails_analysis: List[Dict]
```

Ruff (0.12.2)

Unused method argument: `current_price`

(ARG002)